x-pack/plugins/canvas/canvas_plugin_src/functions/common/__tests__/date.js

```diff
@@ -5,23 +5,12 @@
  */
 
 import expect from '@kbn/expect';
-import sinon from 'sinon';
 import { date } from '../date';
 import { functionWrapper } from '../../../../__tests__/helpers/function_wrapper';
 
 describe('date', () => {
   const fn = functionWrapper(date);
 
-  let clock;
```

Comment:
Tests are failing due to timezones, and I suspect removing this and the stuff in timefilter is what broke them...

Reply:
Yeah, but including it breaks other projects... the tests passed locally.